configure

```diff
@@ -42,3 +43,8 @@ for package in $packages; do
 done
 
 echo ".PHONY: all" >> Makefile
+
+cd LibGAP.jl/test
+sed "s|@@GAPJULIA_DIR@@|${SCRIPTDIR}|g" ../../runtests.jl.in > runtests.jl
+sed "s|@@GAPPATH@@|${GAPPATH}|g" -i runtests.jl
```

Comment:
`-i` is not portable between GNU and BSD sed (not to mention other sed versions.

Instead, just pass multiple scripts to sed, using `-e`. And while at it, might as well drop that `cd`
```
sed -e "s|@@GAPJULIA_DIR@@|${PWD}|g" -e "s|@@GAPPATH@@|${GAPPATH}|g" runtests.jl.in > LibGAP.jl/test/runtests.jl
```